### PR TITLE
Add paper notebooks

### PR DESCRIPTION
**Pull request by richardarsenault**
This PR adds notebooks and required data for reproducing the case studies in the paper to be published. Ideally it will be made available in users' writable-workspace on the platform.

### CODE REVIEW

**Review by tlvu — APPROVED** [review]
Looks good to me.

The location you put your notebooks will not be picked-up by Jenkins, which is intended since you mentioned the notebook will be slow.

As for auto-deploying to the Jupyter env, we can not deploy to `writable-workspace` because we do not want to override user's personal changes. 

Currently the entire Raven repo is cloned under tutorial-notebooks so once you merge this PR, your new paper notebooks will be available there.

File: docs/source/notebooks/paper/tmp.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+
```
[the remark this answers is not in the document]
Comment (richardarsenault) [reply docs/source/notebooks/paper/tmp.txt]: Oops, thanks!

**Comment by tlvu** [comment]
Current location of already existing Raven notebooks on the Jupyter env below.  Your new `paper` folder will be at the same place.

Screenshot from 2022-09-15 15-14-33

**Comment by richardarsenault** [comment]
OK thanks, looks good! It'll be perfect. I just can't merge, can't seem to override the checks like before. I guess we'll wait for them to finish!

**Comment by Zeitsperre** [comment]
Hey @tlvu and @richardarsenault, raven requires a new RavenPy release to fix a few things in the dependencies. I'll see if I can push something either today or tomorrow. I'll keep you both posted.

**Comment by richardarsenault** [comment]
OK thanks! Appreciate it!

**Comment by tlvu** [comment]
Looks like @richardarsenault you can merge now.  Trevor's RavenPy update will be in a separate PR, unless it is needed by this new notebook.

**Comment by tlvu** [comment]
New `paper` notebooks deployed to production Jupyter:

Screenshot from 2022-09-16 09-35-33

**Pull request merged**